lib/rubygems/commands/update_command.rb

```diff
@@ -327,8 +327,26 @@ def which_to_update(highest_installed_gems, gem_names, system = false)
 
   private
 
+  #
+  # Oldest version we support downgrading to. This is the version that
+  # originally ships with the first patch version of each ruby, because we never
+  # test each ruby against older rubygems, so we can't really guarantee it
+  # works. Version list can be checked here: https://stdgems.org/rubygems
+  #
   def oldest_supported_version
-    # for Ruby 2.3
-    @oldest_supported_version ||= Gem::Version.new("2.5.2")
+    @oldest_supported_version ||=
```

Comment:
Would it make sense for future rubies to include info of minimal supported RubyGems into `RbConfig::CONFIG` (or somewhere else) to be able to get rid of this list at some point (and at least do not need to maintain it starting Ruby 3.1 for now)?

Reply:
Maybe? Not really sure.

If we wanted to remove this list we could bump this to use the rubygems version shipped originally with each ruby (here I'm using the version shipped with the `.0` version of each ruby). That version could be extracted from `RbConfig::CONFIG['rubylibdir'] + "/rubygems.rb"` without having to maintain this list.

But given this would require only 1/2 changes per year, it doesn't seem like a big deal?

Reply:
What do you think @simi? Fine as is for now, or should I change anything?

Reply:
Feel free to keep it simple (as if) for now.

Reply:
Alright, let's try it as it is and iterate as we go.